charts/Makefile

```diff
@@ -16,13 +17,19 @@ CHART_VERSION = $(OCP_VERSION)
 CHART_APP_VERSION = $(OCP_VERSION)
 CHART_VALUES_IMAGE_TAG = $(OCP_VERSION)
 CHART_VALUES_IMAGE_TAG_PREFIX = "v"
+
 # Define specific overrides
 ORGREPO_cluster-api-provider-openshift-assisted = https://github.com/openshift-assisted
 KUSTOMIZE_CONFIG_DIRS_cluster-api-provider-openshift-assisted = "bootstrap" "controlplane"
 CHART_VERSION_cluster-api-provider-openshift-assisted = "1.0.0"
 CHART_APP_VERSION_cluster-api-provider-openshift-assisted = $(TAG)
 CHART_VALUES_IMAGE_TAG_cluster-api-provider-openshift-assisted = $(TAG)
 CHART_VALUES_IMAGE_TAG_PREFIX_cluster-api-provider-openshift-assisted = ""
+
+BRANCH_cluster-api-provider-metal3 = main
```

Comment:
I understand that capm3 release cycle will be unrelated to MCE, right? We probably want to link it to some "latest release" rather than "latest (master)". As this project uses github releases, shall we always fetch the latest one?

Reply:
It should be tied to OCP releases, but "latest" would be 4.19 and we don't have a 4.19 image yet so I just stuck with this.

>As this project uses github releases

Does it? I don't see any https://github.com/openshift/cluster-api-provider-metal3/releases

Reply:
:facepalm: I've checked the upstream one, sorry